src/main/resources/templates/help.html

```diff
@@ -15,26 +17,21 @@
 <body>
 <div class="content">
   <header>
-    <img src="images.png" alt="Logo">
+    <img src="../static/images/images.png" alt="Logo">
 
   </header>
   <div class="tab">
-    <button><a href="index.html">About this App</a></button>
+    <button><a href="/">About this App</a></button>
```

Comment:
Fixed href attributes by referencing endpoints defined in Controller as applicable.